jenkins/internal-ci/generic/pr/cortx_deployment/pyutils.groovy

```diff
@@ -237,11 +239,50 @@ pipeline {
                 }
             }
         }
+
+        stage ("Test Utils") {
+            steps {
+                script { build_stage = env.STAGE_NAME }
+                script {
+                        NODE_HOST = sh( script: '''
+                            echo $host | tr ' ' '\n' | head -1 | awk -F["="] '{print $2}' | cut -d',' -f1    
+                        ''', returnStdout: true).trim()
+                        NODE_USER = sh( script: '''
+                            echo $host | tr ' ' '\n' | head -1 | awk -F["="] '{print $3}' | cut -d',' -f1    
+                        ''', returnStdout: true).trim()
+                        NODE_PASS = sh( script: '''
+                            echo $host | tr ' ' '\n' | head -1 | awk -F["="] '{print $4}' | cut -d',' -f1    
+                        ''', returnStdout: true).trim()
+                        def remote = getRemoteMachine(NODE_HOST, NODE_USER, NODE_PASS)
+                        def commandResult = sshCommand remote: remote, command: '''
+                            DATA_POD="$(( kubectl get pods | grep "cortx-data" | awk '{print $1}' ) 2>&1)"
+                            echo $DATA_POD
+                            kubectl exec $DATA_POD --container cortx-hax -- bash -c 'yum install -y wget yum-utils \
```

Comment:
We can use `kubectl exec $(kubectl get pods | awk '/cortx-data/{print $1; exit}')` . This will avoid `DATA_POD` variable declaration and assignment
e.g.
```
[root@ssc-vm-g4-rhev4-1297 ~]# kubectl exec $(kubectl get pods | awk '/cortx-data/{print $1; exit}') --container cortx-hax -- bash -c 'yum install -y wget yum-utils'
Last metadata expiration check: 0:00:38 ago on Fri Apr  1 05:54:47 2022.
Package wget-1.19.5-10.el8.x86_64 is already installed.
Package yum-utils-4.0.21-4.el8_5.noarch is already installed.
Dependencies resolved.
Nothing to do.
Complete!
```

Reply:
Done